File: FSUIKit/Classes/FSTableView.m
//
//  FSTableView.m
//  FSUIKit
//
//  Created by Dongdong Fu on 2022/9/23.
//

#import "FSTableView.h"

@implementation FSTableView

- (void)layoutSubviews {
    [super layoutSubviews];
 
    if (self.layoutedSubviews) {
        self.layoutedSubviews(self);
    }
}

/*
// Only override drawRect: if you perform custom drawing.
// An empty implementation adversely affects performance during animation.
- (void)drawRect:(CGRect)rect {
    // Drawing code
}
*/

@end

File: FSUIKit/Classes/Swift/FSTableView.swift
// FSTableViewS.swift
// Translated from FSTableView.h/m

import UIKit

@objcMembers
public class FSTableView: UITableView {
    
    /// 用来获得tableView reloadData后的时机
    public var layoutedSubviews: ((FSTableView) -> Void)?
    
    public override func layoutSubviews() {
        super.layoutSubviews()
        layoutedSubviews?(self)
    }
}
